pytest-selenium/tests/test_attribution_status.py

```diff
@@ -0,0 +1,30 @@
+from pages.content import Content
+from tests import markers
+
+
+@markers.test_case("C476303")
+@markers.parametrize("book_slug,page_slug", [("college-physics", "preface")])
+@markers.nondestructive
+def test_attribution_collapsed_by_default_expands_when_clicked(
+    selenium, base_url, book_slug, page_slug
+):
+
```

Comment:
No line break here. Capitalize first word in docstring. `Attribution`

Reply:
@m1yag1 Updated